docs_src/microservices/device/Ch-DeviceServiceList.md

```diff
@@ -1,4 +1,4 @@
-# Supported Device Services
+# Supported Device Services List
 
 The following table lists the EdgeX device services and protocols they support.
```

Comment:
For ONVIF and Device Virtual, add links to their documentation. We'll do the same for others as they are added.